### ISSUE TITLE
downtime-replicas not working as expected

### ISSUE THREAD

**Issue by mfilotto** [hjacobs/kube-downscaler#44]
When I set downtime-replicas=2 on a sts:
- the scale down works well 
- but the scale up is stuck to 2 instead of the inital value which is 3

Should I use another annotation ? original replicas ?

**Comment by hjacobs** [comment]
The "original replicas" annotation should be set automatically by `kube-downscaler`, can you check whether this annotation is set? What version of `kube-downscaler` are you running? Can you paste some logs?

**Comment by mfilotto** [comment]
The annotation is set but not to original (3) but 2.
I will try after editing the annotation to 3

**Comment by mfilotto** [comment]
Evene after setting original annotation, I still have a scale up to 2 instead of 3
```2019-04-01 15:02:50,436 INFO: Scaling down StatefulSet dev/zookeeper from 3 to 2 replicas (uptime: always, downtime: Mon-Sun 02:00-02:05 Pacific/Noumea)
2019-04-01 15:05:28,940 INFO: Scaling down StatefulSet dev/zookeeper from 2 to 2 replicas (uptime: always, downtime: Mon-Sun 02:00-02:05 Pacific/Noumea)
2019-04-01 15:08:05,962 INFO: Scaling down StatefulSet dev/kafka from 3 to 2 replicas (uptime: always, downtime: Mon-Sun 02:05-02:35 Pacific/Noumea)
2019-04-01 15:10:43,138 INFO: Scaling down StatefulSet dev/kafka from 2 to 2 replicas (uptime: always, downtime: Mon-Sun 02:05-02:35 Pacific/Noumea)```

**Comment by jasongwartz** [comment]
@mfilotto good catch - not sure if it's 100% the cause but it might be addressed with https://github.com/hjacobs/kube-downscaler/pull/46

I'm guessing based on your comments that you have `downtime-replicas` set to 2, and the original statefulset replicacount is 3?

**Comment by rebbuh** [comment]
@hjacobs I still have the same problem with version 0.18.
kube-downscaler "forgets" the initial number of replicas, i.e. overwrites "downscaler/original-replicas" with the value of "downscaler/downtime-replicas" (> 0) after having scaled down. Therefore, upscaling is not possible after that.

**Comment by hjacobs** [comment]
The best would be to start a PR with a unit test reproducing the problem.

**Comment by rebbuh** [comment]
Sorry, we've been wrong. We made a serious mistake retagging your Docker image. Therefore, we ended up with version 0.16 tagged as 0.18.
With version 0.18, kube-downscaler works perfectly when "downscaler/downtime-replicas" is > 0. :)
Thank you nevertheless!

**Comment by hjacobs** [comment]
@mfilotto is this also fixed for you in v0.18? I would close the issue..
